tests/tool_calling/src/tools.py

```diff
@@ -0,0 +1,459 @@
+import datetime
+import math
+import re
+from llama_stack_client.lib.agents.client_tool import client_tool
+
+
+@client_tool
+def add_two_numbers(a: float, b: float) -> float:
+    """
+    :description: Adds two numbers.
+    :use_case: Use when the user wants to find the sum, total, or combined value of two numbers.
```

Comment:
I'm not sure if you need the `:description:`, `:use_case:` or `:returns:` ( at least according do the client tool decorator docstring). Have you found this to be superior to a plain docstring function description? 

https://github.com/meta-llama/llama-stack-client-python/blob/645d2195c5af1c6f903cb93c293319d8f94c36cc/src/llama_stack_client/lib/agents/client_tool.py#L150-L170